src/widgets/RequisitionItemDetail.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable react/forbid-prop-types */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2020
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View, StyleSheet } from 'react-native';
+
+import { PageInfo } from './PageInfo';
+
+import { getPageInfoColumns } from '../pages/dataTableUtilities';
+
+import { DARKER_GREY, WHITE, SUSSOL_ORANGE } from '../globalStyles';
+
+const RequisitionItemDetailsComponent = ({ item }) => (
+  <View style={localStyles.container}>
+    <PageInfo
+      titleColor={SUSSOL_ORANGE}
+      infoColor={WHITE}
+      columns={getPageInfoColumns('requisitionItemDetail')(item)}
+    />
+  </View>
```

Comment:
this could be a bit nicer memoize the columns and stuff, but with no interactions, it's minimal chance of there being a re-render is minimal so just kept it simple for now

Reply:
agree - there's no real need to memoize this one